src/plugin/memory-webview-main.ts

```diff
@@ -86,7 +87,34 @@ export class MemoryWebview {
         );
     };
 
-    public async show(initialMemory?: Partial<DebugProtocol.ReadMemoryArguments>): Promise<void> {
+    public openCustomDocument(uri: vscode.Uri): vscode.CustomDocument {
+        return {
+            uri,
+            dispose: () => {}
+        };
+    }
+
+    public async resolveCustomEditor(document: vscode.CustomDocument, webviewPanel: vscode.WebviewPanel): Promise<void> {
+        /*
+            memoryReference = debugprotocol.variable.memoryReference
+            displayName = 'memory'
```

Comment:
Out of scope here, but this is a silly label. Better would be e.g. the label of the debug session.